docs/automated-workflows/dagman-scripts.rst

```diff
@@ -42,15 +43,10 @@ file) to be executed, and may not contain spaces. The optional
 the arguments. Both *ExecutableName* and optional *arguments* are case
 sensitive.
 
-A PRE script is commonly used to place files in a staging area for the
-jobs to use. A POST script is commonly used to clean up or remove files
-once jobs are finished running. An example uses PRE and POST scripts to
-stage files that are stored on tape. The PRE script reads compressed
-input files from the tape drive, uncompresses them, and places the
-resulting files in the current directory. The HTCondor jobs can then use
-these files, producing output files. The POST script compresses the
-output files, writes them out to the tape, and then removes both the
-staged files and the output files.
+A PRE script is commonly used to verify inputs for a node job that are produced
+by a parent node, and POST scripts are commonly used to cleanup temporary files
+produced by a node job or to turn a job execution failure into a successful node
+completion.
```

Comment:
I'd remove the temporary files cleanup stuff here (I've never seen that), but emphasize the verification of node success wording.